docs/children-api.md

```
@@ -0,0 +1,90 @@
+# Children API
+
+## 개요
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)